### PR TITLE
Us104726/demo data bookmark

### PR DESCRIPTION
**Pull request by mtjandra**
* Builds on top of https://github.com/BrightspaceHypermediaComponents/activities/pull/123
* Use bookmark query parameter to know what activity to start from
* Use pageSize dynamically

### CODE REVIEW

**Comment by mtjandra** [comment]
My bad, probably should've put  respect page size dynamically commit in separate PR. Feel free to look commit by commit if it helps.

File: demo/d2l-quick-eval/page-handler.js [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
 import { formatActivities } from './activity-handler';
-import chunk from 'lodash-es/chunk';
+
+function getQueryParamOrDefault(
+	relativeUrl,
+	queryParam,
+	defaultValue
+) {
+	const baseUrl = 'http://www.test.com/';
+	const fullUrl = new URL(baseUrl + relativeUrl);
```
Comment (AlexBedley) [inline demo/d2l-quick-eval/page-handler.js]: `URL` actually takes 2 parameters, host and relative, no concatenation required

File: demo/d2l-quick-eval/page-handler.js [hunk 2]
```diff
@@ -1,6 +1,28 @@
 import { parseSortFromUrl, encodeSortState, decodeSortState } from './sort-handler';
 import { formatActivities } from './activity-handler';
-import chunk from 'lodash-es/chunk';
+
+function getQueryParamOrDefault(
+	relativeUrl,
+	queryParam,
+	defaultValue
+) {
+	const baseUrl = 'http://www.test.com/';
```
Comment (AlexBedley) [inline demo/d2l-quick-eval/page-handler.js]: You could use `window.location.origin` to be slightly safer

File: demo/d2l-quick-eval/page-handler.js [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+function setQueryParam(
+	relativeUrl,
+	queryParam,
+	value
+) {
+	const baseUrl = 'http://www.test.com/';
+	const fullUrl = new URL(baseUrl + relativeUrl);
+	fullUrl.searchParams.set(queryParam, value);
+
+	return fullUrl.pathname.substr(1) + '?' + fullUrl.searchParams.toString();
```
Comment (AlexBedley) [inline demo/d2l-quick-eval/page-handler.js]: if you use `fullUrl.search` instead, you don't have to add your own `?`

File: demo/d2l-quick-eval/d2l-quick-eval.html [hunk 4]
```diff
@@ -51,10 +49,10 @@
 		<div class="vertical-section-container fixedSize">
 			<h3>Basic d2l-quick-eval demo</h3>
 			<demo-snippet>
-				<h4>Note: Load more button is expected to fail on second click, but should resolve itself when clicking it again</h4>
+				<h4>Note: Load more button is expected to fail on last click, but should resolve itself when clicking it again</h4>
 				<hr/><br/>
 				<template strip-whitespace>
-					<d2l-quick-eval header-text="Quick Eval" master-teacher href="pages/0?sort=bydate-a" token="whatever"></d2l-quick-eval>
+					<d2l-quick-eval-activities-list header-text="Quick Eval" master-teacher href="pages/?sort=bydate-a" token="whatever"></d2l-quick-eval-activities-list>
```
Comment (AlexBedley) [inline demo/d2l-quick-eval/d2l-quick-eval.html]: Should this still be `<d2l-quick-eval>`? Copy paste error?

File: demo/d2l-quick-eval/demo-data-parser.js [hunk 5]
```diff
@@ -1,7 +1,6 @@
 import { createSortEndpoint } from './sort-handler';
 import { createPageEndpoint } from './page-handler';
 import { formatActivities } from './activity-handler';
-import chunk from 'lodash-es/chunk';
```
Comment (AlexBedley) [inline demo/d2l-quick-eval/demo-data-parser.js]: Let's nuke this from `package.json` then

**Pull request merged**